zulipterminal/config/themes.py

```diff
@@ -103,6 +103,30 @@
     "blue": "zt_blue",
 }
 
+aliases_16_color = [
```

Comment:
Please do migrate this over in a first commit, and ensure everything works (eg. using `make check`) at that point. This will make the second commit smaller and easier to read - and won't have the duplication you have currently.